alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
+                --padding: 6px;
+            }
+            
+            .container {
+                background-color: var(--less-red);
+                border-width: 0;
+                border-left: var(--border-width) solid var(--very-red);
+                border-bottom-right-radius: var(--border-radius);
+                border-top-right-radius: var(--border-radius);
+                padding: var(--padding);
+            }
+            
+            .container [dir='rtl'] {
+		        border-left-width: 0;
+		        border-right: var(--border-width) solid var(--very-red);
```

Comment:
Using the existing React impl as my reference I would say no, because it's just a single `div`.  In order to accomplish certain styles, an inner `div` sometimes becomes necessary, or of course, more complex controls.